src/ProjectService.Models.Db/DbTaskProperty.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ProjectService.Models.Db
+{
+    public class DbTaskProperty
```

Comment:
judging by the class, we came up with some kind of gloom

Reply:
normal